spec/ruby/optional/capi/digest_spec.rb

```diff
@@ -0,0 +1,111 @@
+require_relative 'spec_helper'
+
+require 'ffi'
+
+load_extension('digest')
+
+class RbDigestMetadata < ::FFI::Struct
+  layout :api_version, :int,
+         :digest_len, :size_t,
+         :block_len, :size_t,
+         :ctx_size, :size_t,
+         :init_func, ::FFI::FunctionType.new(:int, [:pointer]),
+         :update_func, ::FFI::FunctionType.new(:void, [:pointer, :pointer, :size_t]),
+         :finish_func, ::FFI::FunctionType.new(:int, [:pointer, :pointer])
+end
```

Comment:
Wondering if this class is used somewhere in the specs.

Reply:
Sorry, nope. I was originally using it verify the contents of the metadata object. I ended up just adding helper methods to the native extension and using those instead. I meant to clean this up. Thanks for pointing it out.